### PR TITLE
Wrap errors on being unable to load configuration

### PR DESCRIPTION
**Pull request by mstoykov**

### CODE REVIEW

**Comment by mstoykov** [comment]
This should improve situations such as https://community.k6.io/t/sample-scripts-unexpected-end-of-json/3940/2

File: cmd/config.go [hunk 1]
```diff
@@ -137,10 +137,14 @@ func readDiskConfig(globalState *globalState) (Config, error) {
 
 	data, err := afero.ReadFile(globalState.fs, globalState.flags.configFilePath)
 	if err != nil {
-		return Config{}, err
+		return Config{}, fmt.Errorf("Couldn't load configuration from %q:%w", globalState.flags.configFilePath, err)
```
Comment (na--) [inline cmd/config.go:140]:
```suggestion
		return Config{}, fmt.Errorf("couldn't load configuration from %q: %w", globalState.flags.configFilePath, err)
```

File: cmd/config.go [hunk 2]
```diff
@@ -137,10 +137,14 @@ func readDiskConfig(globalState *globalState) (Config, error) {
...
 	}
 	var conf Config
-	return conf, json.Unmarshal(data, &conf)
+	err = json.Unmarshal(data, &conf)
+	if err != nil {
+		return Config{}, fmt.Errorf("Couldn't parse configuration from %q:%w", globalState.flags.configFilePath, err)
```
Comment (na--) [inline cmd/config.go:145]:
```suggestion
		return Config{}, fmt.Errorf("couldn't parse configuration from %q: %w", globalState.flags.configFilePath, err)
```

File: cmd/config.go [hunk 3]
```diff
@@ -137,10 +137,14 @@ func readDiskConfig(globalState *globalState) (Config, error) {
 
 	data, err := afero.ReadFile(globalState.fs, globalState.flags.configFilePath)
 	if err != nil {
-		return Config{}, err
+		return Config{}, fmt.Errorf("couldn't load configuration from %q:%w", globalState.flags.configFilePath, err)
```
Comment (na--) [inline cmd/config.go:140]:
```suggestion
		return Config{}, fmt.Errorf("couldn't load configuration from %q: %w", globalState.flags.configFilePath, err)
```

**Review by na-- — APPROVED** [review]

**Review by olegbespalov — APPROVED** [review]

**Pull request merged**